README.md

```diff
@@ -4,60 +4,150 @@
+- `sourcecred score`
 
-[MetaGame's Cred Instance]: https://github.com/MetaFam/TheSource
+## Contributing Guidelines
+
+If you'd like to contribute to the codebase, we ask you to follow the following
+steps:
+
+### 1. Drop by [our Discord].
+
+Come to the #intros channel and introduce yourself. Let us know that you're
+interested in helping out. We're friendly and will be happy to help you get
+oriented.
+
+### 2. Read our [Contributing Guidelines].
+
+We pride ourself on tidy software engineering; part of how we do that is by
+splitting our changes up into many small, atomic commits, each of which are
+easy to review. If you'd like to work alongside us, we ask you to adopt our
+practices.
+
+### 3. Find an issue to work on.
+
+You can check out the issues marked [contributions welcome], or ask in the
+Discord's #programming channel if anyone has something you can contribute to.
+
+[our Discord]: https://sourcecred.io/discord
+[Contributing Guidelines]: ./contributing.md
+[contributions welcome]: https://github.com/sourcecred/sourcecred/issues?q=is%3Aopen+is%3Aissue+label%3Acontributions-welcome
 
 ## Development Setup
 
+### Dependencies
+
   - Install [Node] (tested on v12.x.x and v10.x.x).
   - Install [Yarn] (tested on v1.7.0).
-  - Create a [GitHub API token]. No special permissions are required.
   - For macOS users: Ensure that your environment provides GNU
     coreutils. [See this comment for details about what, how, and
     why.][macos-gnu]
 
 [Node]: https://nodejs.org/en/
 [Yarn]: https://yarnpkg.com/lang/en/
-[GitHub API token]: https://github.com/settings/tokens
 [macos-gnu]: https://github.com/sourcecred/sourcecred/issues/698#issuecomment-417202213
 
+If you want to work on the GitHub plugin, you should
+create a [GitHub API token]. No special permissions are required.
+
+[GitHub API token]: https://github.com/settings/tokens
+
+Then, set it in your environment:
+```Bash
+export SOURCECRED_GITHUB_TOKEN=1234....asdf
+```
+
+### Building SourceCred Locally
+
 You'll still need to create a GitHub token to use as an environment variable (shown later). First, run the following commands to clone and build SourceCred:
 
 ```Bash
 git clone https://github.com/sourcecred/sourcecred.git
 cd sourcecred
-yarn install
+yarn
 yarn build
-export SOURCECRED_GITHUB_TOKEN=YOUR_GITHUB_TOKEN
 ```
 
-Once that's been setup, you can start running SourceCred development commands.
-In master, the commands are currently in flux. This README will be updated when we have a stable CLI again.
+### Using A Modified Backend
+
+You'll likely want to test out your modified version of SourceCred on an
+instance you're familiar with. A convenient way to do that is to create an
+alias for your altered version of SourceCred. Here's an example of how to do
+so:
+
+```Bash
+SC_REPOSITORY_DIR=`pwd`
+alias scdev=node "$SC_REPOSITORY_DIR"/bin/sourcecred.js
+cd $MY_SC_INSTANCE
+# Run the `sourcecred go` command, in your instance, using your modified code.
+scdev go
+```
```

Comment:
might be better to use the NPM package here. Either `yarn global add sourcecred` / `npm i -g sourcecred`, or if its just running it one-off they dont need to install anything and can do `npx sourcecred go`

Reply:
This section specifically has instructions for using a locally-modified copy of SourceCred.